package.json

```diff
@@ -32,6 +34,21 @@
   "dependencies": {
     "@parcel/core": "^2.8.3",
     "@parcel/packager-ts": "^2.8.3",
-    "@parcel/transformer-typescript-types": "^2.8.3"
+    "@parcel/transformer-typescript-types": "^2.8.3",
+    "@preconstruct/cli": "^2.6.4"
+  },
+  "preconstruct": {
+    "packages": [
+      "packages/!(react-resizable-panels-website)"
+    ],
+    "globals": {
+      "react": "React"
+    },
+    "exports": {
+      "importConditionDefaultExport": "default"
+    },
+    "___experimentalFlags_WILL_CHANGE_IN_PATCH": {
```

Comment:
the name is scary since we just shipped this as a new feature but we don't intend to break this, if we promote this to a stable feature, rename this or anything then `preconstruct fix` will just become capable of migrating this to the new format